### PR TITLE
log application id when uploading and bundle id when deploying

### PR DESCRIPTION
**Pull request by aronatkins**

### CODE REVIEW

File: R/deployApp.R [hunk 1]
```diff
@@ -189,7 +189,8 @@ deployApp <- function(appDir = getwd(),
 
   if (upload) {
     # create, and upload the bundle
-    withStatus("Uploading application bundle", {
+    withStatus(paste("Uploading bundle for application:",
+                     application$id) {
```
Comment (kippandrew) [inline R/deployApp.R]: Bug: Missing closing `)`

Comment (kippandrew) [inline R/deployApp.R]: Correction: missing a colon

Comment (aronatkins) [inline R/deployApp.R]: The paren is closed later (`withStatus` takes an expression). The colon is there. What am I missing?

Comment (kippandrew) [inline R/deployApp.R]: Correction: I mean comma. lol

**Pull request merged**